### ISSUE TITLE
Theano cannot find cuDNN on win7

### ISSUE THREAD

**Issue by oiale** [Theano/Theano#4545]
Hello,

I can't get cudnn working with Theano 0.9.0.dev1 on win7 x64 with gtx 660. I downloaded cudnn v5 and unpacked files over cuda installation (include > include, bin > bin and so on). However, when I check it with
python -c "from theano.sandbox.cuda.dnn import dnn_available as da; print(da() or da.msg)" 
I get the following error:
Can not compile with cuDNN. We got this error:
b''

Is there a way to fix it or at least get more useful output on the error?

**Comment by nouiz** [comment]
You can set a break point by using pdb in the file theano/sandbox/cuda/__init__.py;

Add this the line "import pdb;pdb.set_trace()" at the start of the function dnn_available(). This will allow you to investigate what is going on. If you have questions, don't hesitate.

**Comment by genichyar** [comment]
I had the same problem. It occured when compiler_bindir had spaces. For example, "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin".

My solution is replace
```
params.extend(['--compiler-bindir',
                               config.nvcc.compiler_bindir])
```
to
```
params.extend(['--compiler-bindir',
                               path_wrapper + config.nvcc.compiler_bindir + path_wrapper])
```
in `theano/sandbox/cuda/__init__.py`

With this fix cudnn works well.

**Comment by nouiz** [comment]
Can you make a PR with this?

**Comment by lamblin** [comment]
Fixed in #4923, and we're using the new back-end now anyway.
